rasa/shared/core/slots.py

```diff
@@ -149,8 +149,23 @@ def persistence_info(self) -> Dict[str, Any]:
             "influence_conversation": self.influence_conversation,
         }
 
+    def fingerprint(self) -> Text:
+        """Returns a unique hash for the slot which is stable across python runs.
+
+        Returns:
+            fingerprint of the slot
+        """
+        data = [
```

Comment:
I'm not sure this is enough. What about things like `influence_conversation`? and for subclasses like `CategoricalSlot` we have things like `self.values`.

Reply:
I've changed the fingerprinting by using `persistence_info` which should contain the extra bits from certain subclassess (such as `self.values` for `CategoricalSlot`)